.gitlab-ci.yml

```diff
@@ -26,7 +26,7 @@ build_docker:
   before_script:
     - mkdir -p ${PIP_CACHE_DIR}
     - apt-get update
-    - apt-get install -y swig python3-opencv
+    - apt-get install -y swig python3-opencv libosmesa6-dev patchelf
```

Comment:
Necessary binaries to run the tests with environment using the `mujoco` library.

Reply:
You should add them in the setup documentation.